spec/models/user_spec.rb

```diff
@@ -48,4 +46,13 @@
       end
     end
   end
+  describe 'being deactivated' do
+    let(:future_assignment) { create :assignment, start_date: Date.tomorrow }
+    it 'destroys future assignments for users' do
+      user = future_assignment.user
+      user.active = false
+      user.save
+      expect(user.assignments).to eq []
```

Comment:
could you use `be_empty` on this?

Reply:
Interesting, though, that an empty `ActiveRecord::Relation` actually _does_ `eq []`